src/xstate/User/userMachine.ts

```diff
@@ -0,0 +1,59 @@
+import {createMachine, assign} from 'xstate';
+import {TUser} from './types';
+
+type TLoginValues = {
+  email: string;
+  password: string;
+};
+
+export const userMachine = createMachine(
+  {
+    tsTypes: {} as import('./userMachine.typegen').Typegen0,
```

Comment:
Ojeju, tak trzeba? Ogólnie do wszystkich typ rzutowań na typy, coś nie ten tegens jest jak oznaczasz pusty obiekt na coś co ma properties

Reply:
tbh chuj wie to było z tego co ci mówiłem na yt i tą linijkę jak i plik samo generuje jakaś wtyczka do vsc